src/frontend/komponenter/Fagsak/Vilkårsvurdering/GeneriskVilkår/VilkårSkjemaContext.ts

```diff
@@ -27,18 +29,18 @@ export interface IVilkårSkjemaContext {
 
 export interface VilkårSkjemaContextValue<T extends IVilkårSkjemaContext> {
     skjema: ISkjema<T, IBehandling>;
-    lagreVilkår: () => void;
+    lagreVilkår: (onSuccess?: () => void) => void;
     lagrerVilkår: boolean;
-    slettVilkår: (personIdent: string, vilkårId: number) => void;
+    slettVilkår: (personIdent: string, vilkårId: number, onSuccess?: () => void) => void;
     sletterVilkår: boolean;
     feilmelding: string;
+    nullstillSkjema: () => void;
```

Comment:
Hadde det gitt mening å kalle denne `tilbakestillSkjemaTilLagredeVerdier` eller noe i den duren? For det er jo eeeegentlig det man gjør.

Reply:
Ja, er enig at `nullstill` ikke helt gir mening. Men det fine med det er at det gjenspeiler navngivingen i skjemahooken vi bruker, der `nullstill` på feltet også betyr å stille det tilbake til lagret verdi fra serveren. Du kan jo se hvor mye du ønsker å endre det, jeg har ingen sterke meninger der 😊